src/Attribute/Suppress.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\EventSourcing\Attribute;
+
+use Attribute;
+use InvalidArgumentException;
+use Patchlevel\EventSourcing\Aggregate\AggregateChanged;
+
+use function is_string;
+
+#[Attribute(Attribute::TARGET_CLASS)]
+class Suppress
```

Comment:
Im not sure if i like the idea itself but if the do this i think the naming can be improved:
`SuppressMissingApplyMethods` would be more explaining i think